packages/sitecore-jss-nextjs/src/middleware/personalize-middleware.ts

```diff
@@ -42,13 +37,6 @@ export type PersonalizeMiddlewareConfig = MiddlewareBaseConfig & {
    * Configuration for your Sitecore CDP endpoint
    */
   cdpConfig: CdpServiceConfig;
-  /**
-   * function to resolve point of sale for a site
-   * @param {Siteinfo} site to get info from
-   * @param {string} language to get info for
-   * @returns point of sale value for site/language combination
-   */
-  getPointOfSale?: (site: SiteInfo, language: string) => string;
```

Comment:
@ambrauer I had a call with Cloud SDK team and they said that there shouldn't be a way to customize that, since it's tied to siteName. In any case I asked them to provide a response here
cc @sc-nikolaoslazaridis

Reply:
Sounds like we are good here